packages/docusaurus-plugin-content-docs/src/metadata.ts

```diff
@@ -128,6 +128,11 @@ export default async function processMetadata({
     routePath,
   ]);
 
+  const latestPermalink =
+    version && version !== versioning.latestVersion
+      ? normalizeUrl([baseUrl, routeBasePath])
+      : undefined;
```

Comment:
This is almost what we want :)

on D2 website, we have a docs `homePageId`, so the root is `/docs`

```
        docs: {
          homePageId: 'introduction',
          path: 'docs',
```

But there are sites that do not have any docs home, it's the case for Watchman:
- https://facebook.github.io/watchman/docs => 404
- https://facebook.github.io/watchman/docs/install => this is the watchman home

For sites like Watchman without a doc at the root, we'd rather make sure that we don't link to a path that does not exist